compilation fails on ruby <2.7

the gem claims to support ruby 2.5+, but on ruby 2.5/2.6 it fails on missing dependencies
```
make: *** No rule to make target '/opt/hostedtoolcache/Ruby/2.6.8/x64/include/ruby-2.6.0/ruby/assert.h', needed by 'escape.o'.  
Stop.
```

actually, the gem doesn't compile extensions at all because there's a missing line in cgi.gemspec
```
spec.extensions    = ["ext/cgi/escape/extconf.rb"]
```

I think it's a bug or is it intentional?

--- a/cgi.gemspec
+++ b/cgi.gemspec
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
     `git ls-files -z 2>/dev/null`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
+  spec.extensions    = ["ext/cgi/escape/extconf.rb"]
   spec.bindir        = "exe"
   spec.executables   = []
   spec.require_paths = ["lib"]

